packages/venia-ui/.storybook/config.js

```diff
@@ -1,8 +1,26 @@
-import { configure } from '@storybook/react';
+import React from 'react';
+import { configure, addDecorator } from '@storybook/react';
+import { AppContextProvider } from '../lib/components/App';
+import { Adapter } from '@magento/venia-drivers';
+import store from '@magento/venia-concept/src/store';
+import backendUrl from './backendUrl';
+import '@magento/venia-concept/src/index.css';
 
 function loadStories() {
-    const context = require.context('../src', true, /__stories__\/.+\.js$/);
+    const context = require.context('../lib', true, /__stories__\/.+\.js$/);
     context.keys().forEach(context);
 }
 
+const apiBase = new URL('/graphql', backendUrl).toString();
+
+addDecorator(storyFn => (
+    <Adapter
+        apiBase={apiBase}
```

Comment:
Note to team: this may need to be updated alongside #1966 .

Reply:
Does this still need to get updated or can we resolve the comment? That PR has been merged.

Reply:
Looks like we're good 👍